### DISCUSSION TITLE
Feature request: Change the search engine used in the start menu

### DISCUSSION THREAD

**Discussion by rp1231** [valinet/ExplorerPatcher#1028] [Ideas]
Would it be possible to replace the search engine used in the start menu?
 
![sidekick_kFc9Q6znvc](https://user-images.githubusercontent.com/47380919/157116399-e8a9cdde-18ca-4526-9664-ba28f71a71c5.png)
 
Thanks

**Comment by rp1231** [comment]
Quoted from the reply from @Shaughn-david
> I think you are supposed to use discussions.
> Anyway, this sounds like a good idea since it is hard to find what I am looking for when I use bing.

**Comment by Poopooracoocoo** [comment]
Might also wanna check out BeautySearch. The developer of that has gone a bit over modding Windows Search, which by the way, is separate to the start menu. 

For a workaround, you can install something like Wox, or you can use EdgeDeflector and stuff.
